### ISSUE TITLE
aarch64-musl builder needs update for x86_64 kernel-libc-headers.

### ISSUE THREAD

**Issue by pullmoll** [voidlinux/void-packages#5904]
It looks like we have another case of aarch64-musl lacking an update.

I don't really understand why these dependencies are not automatically rebuilt. Is it because they are resolved through cross-vkpg-dummy?

**Comment by chneukirchen** [comment]
```
index: added `kernel-libc-headers-4.9.8_2' (x86_64).
index: added `kernel-libc-headers-4.9.8_2' (x86_64-musl).
```

**Comment by pullmoll** [comment]
Oops. It's really the cross compilers which need an update, i.e. cross-aarch64-linux-gnu and (more importantly) cross-aarch64-linux-musl, because the patched kernel-libc-headers-4.9.8 are there.

This is why the aarch64-musl builder currently fails to build NetworkManager.

**Comment by chneukirchen** [comment]
Done.
